lib/server/booterror.js

```diff
@@ -29,7 +29,7 @@ function bootError(env, ctx) {
       return '<dt><b>' + obj.desc + '</b></dt><dd>' + message.replace(/\\n/g, '<br/>') + '</dd>';
     }).join(' ');
 
-    res.render('error.html', {
+    res.status(500).render('error.html', {
```

Comment:
https://expressjs.com/en/api.html#res.status

This returns a `500 error` http status in the header, rather than the default `200 ok`

Reply:
Good idea, I love it.